windse/SolverManager.py

```diff
@@ -212,11 +212,46 @@ def ChangeWindAngle(self,inflow_angle):
     #     f.close()
 
 
+    def EvaluateObjective(self):
+        self.fprint("Evaluating Objective Data",special="header")
+        start = time.time()
 
+        first_call = True
+        if self.J_saved:
+            first_call = False
 
+        ### Convert to list ###
+        if  not isinstance(self.objective_type,list):
+            self.objective_type = [self.objective_type]
 
+        ### Iterate over objectives ###
+        obj_list = [self.simTime]
+        for objective in self.objective_type:
+            self.objective_func = obj_funcs.objectives_dict[objective]
+            obj_list.append(self.objective_func(self,(self.iter_theta-self.problem.dom.inflow_angle),first_call=first_call))
+        J = obj_list[1]
 
+        ### Flip the sign because the objective is minimized but these values are maximized
+        for i in range(1,len(obj_list)):
+            obj_list[i] = -obj_list[i]
```

Comment:
This function looks good, but I think there might be instances where we don't want to flip the sign.  I see it's just a cosmetic thing, but it might be clearer to record what the optimizer is actually accessing.  What do you think?